console/console-init/ui/src/Routes.tsx

```diff
@@ -92,10 +92,10 @@ export const AppRoutes = () => (
     <LazyRoute
       path="/iot-projects/:namespace/:projectname/devices/:deviceid/:subList"
       getComponent={getDeviceDetailPage}
-      />
+    />
     <LazyRoute
       path="/iot-projects/:namespace/:projectname/:sublist"
-      getComponent={getIoTProjectDetailPage}
+      getComponent={getProjectDetailPage}
```

Comment:
I will suggest name as IoTProjectDetail as it will create confusion in messaging and iot projects